bitnami/keycloak/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: keycloak
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/keycloak
   - https://github.com/keycloak/keycloak
-version: 13.1.3
+version: 13.1.4
```

Comment:
Hi @nkreiger,

As this is PR introduces a new feature, could you please bump a minor version? (13.2.0)

Reply:
sure!